Tests/SentryTests/Protocol/SentryIdTests.swift

```diff
@@ -22,6 +22,14 @@ class SentryIdTests: XCTestCase {
     func testInit() {
         XCTAssertNotEqual(SentryId(), SentryId())
     }
+    
+    func testInitFromString() {
+        measure {
+            let string = "abcdefabcdefabcdefabcdefabcdefab"
+            let sentryId = SentryId(uuidString: "abcdefabcdefabcdefabcdefabcdefab")
+            XCTAssertEqual(sentryId.sentryIdString, string)
+        }
```

Comment:
m: `measure` don't work for CI.
```suggestion
        let string = "abcdefabcdefabcdefabcdefabcdefab"
        let sentryId = SentryId(uuidString: "abcdefabcdefabcdefabcdefabcdefab")
        XCTAssertEqual(sentryId.sentryIdString, string)
```

Reply:
ah yeah, forgot to remove it after running the experiment